include/fbgemm/QuantUtils.h

```diff
@@ -131,7 +131,7 @@ FBGEMM_API std::int64_t
 SaturatingRoundingMulWithShift(std::int32_t a, std::int32_t b, int right_shift);
 
 template <typename T>
-FBGEMM_API T Requantize(
+T Requantize(
```

Comment:
These are needed for shared library build on Linux. What problems do they create on windows?

Reply:
They are not needed because:
1. They are the in the header file so they will be visible at the user side.
2. They have the implementation, not just declarations.